appinventor/appengine/war/Ya.css

```diff
@@ -1287,6 +1287,108 @@ path.ode-SimpleMockMapFeature-selected {
 /* All dialog boxes */
 /* would pop more with a white border around their normal green border */
 
+/* Start of NoProjectsDialogBox. */
+.ode-noDialogDiv {
+  box-shadow: 0px 0px 10px gray;
+  min-height: 60%;
```

Comment:
This still is resulting in the dialog div being rather large on my screen with a lot of whitespace. It might make sense to set this to a fixed value.